__tests__/materials/material.mutation.test.js

```diff
@@ -1,427 +1,415 @@
-                  }
-                  description {
-                    lang
-                    value
-                  }
-                  purpose
+  beforeAll(async () => {
+    operations = await setupApp();
+  });
+
+  it('#1 should add material to database', async () => {
+    const res = await operations.mutate({
+      mutation: gql`
+        mutation($material: MaterialInput!) {
+          addMaterial(material: $material) {
+            ... on Material {
+              _id
+              name {
+                lang
+                value
+              }
+              description {
+                lang
+                value
+              }
+              purpose
 
-                  colors {
-                    code
-                    name {
-                      lang
-                      value
-                    }
-                    simpleName {
-                      lang
-                      value
-                    }
-                    available
-                    images {
-                      large
-                      medium
-                      small
-                      thumbnail
-                    }
-                  }
-                  additionalPrice {
-                    currency
-                    value
-                  }
-                  available
+              colors {
+                code
+                name {
+                  lang
+                  value
                 }
-                ... on Error {
-                  message
-                  statusCode
+                simpleName {
+                  lang
+                  value
                 }
+                available
+                images {
+                  large
+                  medium
+                  small
+                  thumbnail
+                }
+              }
+              additionalPrice {
+                currency
+                value
               }
+              available
             }
-          `,
-        })
-        .catch(e => e);
-      const newMaterial = res.data.addMaterial;
-      materialId = newMaterial._id;
+            ... on Error {
+              message
+              statusCode
+            }
+          }
+        }
+      `,
+      variables: { material },
+    });
 
-      expect(newMaterial).toBeDefined();
-      expect(newMaterial).toHaveProperty(
-        'name',
-        material.name.map(item => ({ ...languageTypeName, ...item }))
-      );
-      expect(newMaterial.name).toBeInstanceOf(Array);
+    const addedMaterial = res.data.addMaterial;
+    materialId = res.data.addMaterial._id;
+    expect(addedMaterial).toBeDefined();
 
-      expect(newMaterial).toHaveProperty(
-        'description',
-        material.description.map(item => ({ ...languageTypeName, ...item }))
-      );
-      expect(newMaterial.description).toBeInstanceOf(Array);
+    expect(addedMaterial).toHaveProperty(
+      'name',
+      material.name.map(item => ({
```

Comment:
Please, simplify it to the `material.name` because mapping is useless now due to the returning the same object